lib/knapsack_pro/hooks/queue.rb

```diff
@@ -0,0 +1,21 @@
+module KnapsackPro
+  module Hooks
+    class Queue
+      class << self
+        attr_reader :after_subset_queue
+
+        def after_subset_queue(&block)
+          @after_subset_queue ||= block
+        end
```

Comment:
Yep, I tried that here https://github.com/KnapsackPro/rails-app-with-knapsack_pro/pull/6/files but there is another problem described here https://github.com/KnapsackPro/knapsack_pro-ruby/issues/40#issuecomment-303519352